roles/delete_nodepools/tasks/main.yml

```diff
@@ -11,17 +11,19 @@
 
 - name: Use Terraform output in variables
   ansible.builtin.set_fact:
```

Comment:
I think we don't need this step, it's easier to understand if you use terraform_run directly without storing the outputs in intermediate variables